swift/test/Ice/admin/AllTests.swift

```diff
@@ -239,20 +239,20 @@ func allTests(_ helper: TestHelper) throws {
             "Prop3": "3",
         ]
 
-        let com = try factory.createCommunicator(props)!
-        let obj = try com.getAdmin()!
-        let pa = try checkedCast(prx: obj, type: Ice.PropertiesAdminPrx.self, facet: "Properties")!
+        let com = try await factory.createCommunicator(props)!
+        let obj = try await com.getAdmin()!
+        let pa = try await checkedCast(prx: obj, type: Ice.PropertiesAdminPrx.self, facet: "Properties")!
 
         //
         // Test: PropertiesAdmin::getProperty()
         //
-        try test(pa.getProperty("Prop2") == "2")
-        try test(pa.getProperty("Bogus") == "")
+        try await test(pa.getProperty("Prop2") == "2")
```

Comment:
How this works?

Shouldn't it be `try test(await pa.getProperty("Prop2") == "2")`

Reply:
Nope, in Swift you only need one await call and it goes all the way left before any expressions.